packages/components/ResultData.tsx

```diff
@@ -0,0 +1,36 @@
+import { Flex, Grid } from '@chakra-ui/react';
+
+// const listCandidate = ['1 colocado', '2 colocado', '3 colocado'];
+
+export default function Display() {
+  return (
+    <Grid
```

Comment:
I believe you should remove this grid from here and use a flex.

For now,  Just create a flex that will render N cards, even if it adds a scroll in the page, and modify your component to already use a map to render some mocked "cards"